subscription_manager.go

```diff
@@ -532,15 +537,17 @@ func subscribeMessageWorker(m *SubscriptionManager) {
 		select {
 		case <-m.exitSubscriptionManager:
 			m.pubnub.Config.Log.Println("subscribeMessageWorker context done")
-			m.exitSubscriptionManager = nil
-			break
+			// m.exitSubscriptionManagerMutex.Lock()
```

Comment:
Are these intentionally left in for a reason?

Reply:
will remove